src/app/hierarchy/hierarchy.demo.ts

```diff
@@ -52,7 +52,15 @@ export class HierarchyDemoComponent implements OnInit {
             <p class="subheading">{{Position}}</p>
             <p class="micro">{{EmploymentType}}</p>
            </div>
-
+           {{#menu}}
+            <button class="btn-actions btn-icon" type="button" data-init="false" id="btn-{{id}}">
```

Comment:
Use `soho-button`?

Reply:
Can't use soho-button here unfortunately. This template stub is used internally by the hierarchy control and is outside of Angular scope (embedded into the document). Would be nice if angular template could have been leveraged somehow. But this control was originally wrote pre-angular.

Reply:
Good point - missed the context of the code as I can only see a snippet.  

It's possible to fix the hierarchy component to allow nested components (datagrid does this, optionally, for editors and formatters).

Reply:
Possibly? I think that would be doable. But if every leaf essentially had an underlying Angular component, might get a little heavy maybe? Could give it a shot though

Reply:
@krishollenbeck is this ready to be merged?

Reply:
@pwpatton, Yeah it's (was) ready